src/polytopes/ngon.jl

```diff
@@ -80,6 +80,33 @@ function measure(t::Triangle{3})
   norm((B - A) × (C - A)) / 2
 end
 
+function circumcenter(t::Triangle{2})
+  A, B, C = t.vertices
+  sin2A = sin(2 * ∠(C, A, B))
+  sin2B = sin(2 * ∠(A, B, C))
+  sin2C = sin(2 * ∠(B, C, A))
+  sines = [sin2A, sin2B, sin2C]
+  x₁, y₁ = coordinates(A)
+  x₂, y₂ = coordinates(B)
+  x₃, y₃ = coordinates(C)
+  x = [x₁, x₂, x₃]
+  y = [y₁, y₂, y₃]
+  Point((x ⋅ sines) / sum(sines), (y ⋅ sines) / sum(sines))
+end
+
+function circumcenter(t::Triangle{3})
+  A, B, C = t.vertices
+  vAB = B - A
+  vAC = C - A
+  mAB = coordinates(A + vAB/2)
+  mAC = coordinates(A + vAC/2)
+  n⃗ = normal(t)
+  offset = coordinates(A) ⋅ n⃗
```

Comment:
I would try to use single letter for all variables given that the function is mostly made of single letter variables.

Reply:
Hmm.. I like `vAB`: vector from A to B. And `mAB` is for middle.

Reply:
I referred to the variable `offset`, it is kind of an outlier in the code when all variables are short.